specs/credentialmanagement/index.src.html

```diff
@@ -927,18 +975,6 @@ <h4 id="request-credential">
       </ol>
     </li>
     <li>
-      Let <var>type</var> be lowest common ancestor interface of the types
-      referenced in <var>request</var>'s {{CredentialRequest/types}} property.
-
-      ISSUE(w3c/webappsec#289): This is terribly hand-wavey. The intent is
-      clear, but I need to do the work to walk through the list of DOMStrings
-      and convert them to interfaces and etc. Busywork for later.
```

Comment:
This is where my knowledge of the inner working of the browser are limited so I apologize in advance if I'm way off.

The way I envisioned this working was something like this:

1. The site owner provides the definition of the custom credential via a script.
1. The APi uses that definition to create a new instance of that credential (populated with the data stored in the credential store).
1. This credential is not returned to the caller without user mediation.